documentation/release-notes/topics/version_4.rst

```diff
@@ -2,10 +2,61 @@
 Version 4
 #########
 
+*******************
+Hyperview 4.4 (TBD)
+*******************
+This section covers significant changes and bug fixes in Hyperview 4.4.x since version 4.3.x
+
+.. important::
+   1. This release has changes and improvements to the Data Collector software. We strongly recommend updating your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+   2. Support for the *Windows* version of the Data Collector will end on *January 31, 2025*. Customers are encouraged to switch to the Linux version before then. Detailed :ref:`installation instructions <Setting-up-data-collectors-doc>` are in the product documentation.
+   3. `CentOS Linux 7 End of Life, is June 30, 2024, <https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/>`_. With that, installing the Data Collector will not be supported on this version of CentOS. If you are using this version of Linux, you must update to a :ref:`supported version <linux-prerequisites>` to use the latest version of the Data Collector.
+
+
+===============================
+New Feature - Business Entities
+===============================
+Business Entities allow customers to track customers, partners, and department associations to assets, connections and circuits. Business Entities can have contacts and addresses as well as:
+
+- Access Control
+- Change Log
+- Custom properties
```

Comment:
For consistency, this should be "Custom Properties" or they should all be formatted like this.

Reply:
Fixed! Thank you.